0001-remove-spandata.md

```diff
@@ -21,12 +20,21 @@ I'd like to propose getting rid of SpanData and `tracer.recordSpanData()` and re
 
 From https://github.com/open-telemetry/opentelemetry-specification/issues/71: If the underlying SDK automatically adds tags to spans such as thread-id, stacktrace, and cpu-usage when a span is started, they would be incorrect for out of band spans as the tracer would not know the difference between in and out of band spans. This can be mitigated by indicating that the span is out of band to prevent attaching possibly incorrect information.
 
-https://github.com/open-telemetry/opentelemetry-specification/issues/96 discusses the possibility of allowing SpanData to support lazy fields, preventing allocations until it is read by the exporter. With span options, I don’t believe we would be able to get a fully lazy span, although events and logs could easily be made lazy in the future.
+https://github.com/open-telemetry/opentelemetry-specification/issues/96 discusses the possibility of allowing SpanData to support lazy fields, preventing allocations until it is read by the exporter. With span options, I don’t believe we would be able to get a fully lazy span, but we can get really close, especially by adding lazily evaluated events and logs. Some discussion should be had about the laziness questions in the Open Questions section.
```

Comment:
I feel it's worth stating explicitly that the OpenTracing Span FinishOptions includes bulk logs for this reason--somewhere. I feel that "really close" is a loose way of saying the requirement is "effectively satisfied".